2025-03-05 09:12:07,925 INFO    MainThread:25332 [wandb_init.py:setup_run_log_directory():647] Logging user logs to d:\IIT Madras\SEM 2\Deep Learning\Assignment-1-DL\wandb\run-20250305_091207-1hy195yf\logs\debug.log
2025-03-05 09:12:07,926 INFO    MainThread:25332 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to d:\IIT Madras\SEM 2\Deep Learning\Assignment-1-DL\wandb\run-20250305_091207-1hy195yf\logs\debug-internal.log
2025-03-05 09:12:07,926 INFO    MainThread:25332 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000020BD4213BD0>
2025-03-05 09:12:07,926 INFO    MainThread:25332 [wandb_init.py:init():761] calling init triggers
2025-03-05 09:12:07,926 INFO    MainThread:25332 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-05 09:12:07,926 INFO    MainThread:25332 [wandb_init.py:init():784] starting backend
2025-03-05 09:12:07,926 INFO    MainThread:25332 [wandb_init.py:init():788] sending inform_init request
2025-03-05 09:12:07,928 INFO    MainThread:25332 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-03-05 09:12:07,928 INFO    MainThread:25332 [wandb_init.py:init():803] backend started and connected
2025-03-05 09:12:07,930 INFO    MainThread:25332 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-05 09:12:07,941 INFO    MainThread:25332 [wandb_init.py:init():896] updated telemetry
2025-03-05 09:12:07,993 INFO    MainThread:25332 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-05 09:12:09,247 INFO    MainThread:25332 [wandb_init.py:init():995] starting run threads in backend
2025-03-05 09:12:09,500 INFO    MainThread:25332 [wandb_run.py:_console_start():2377] atexit reg
2025-03-05 09:12:09,500 INFO    MainThread:25332 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-05 09:12:09,500 INFO    MainThread:25332 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-05 09:12:09,500 INFO    MainThread:25332 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-05 09:12:09,501 INFO    MainThread:25332 [wandb_init.py:init():1037] run started, returning control to user process
2025-03-05 09:12:10,181 INFO    MainThread:25332 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-05 09:12:10,181 INFO    MainThread:25332 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-05 09:12:13,157 INFO    MainThread:25332 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-05 09:12:13,161 INFO    MainThread:25332 [pyagent.py:run():322] Starting sweep agent: entity=None, project=None, count=None
2025-03-05 09:12:14,321 WARNING MsgRouterThr:25332 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "C:\Users\Rajat\AppData\Roaming\Python\Python311\site-packages\wandb\sdk\interface\router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rajat\AppData\Roaming\Python\Python311\site-packages\wandb\sdk\lib\sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rajat\AppData\Roaming\Python\Python311\site-packages\wandb\sdk\lib\sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Rajat\AppData\Roaming\Python\Python311\site-packages\wandb\sdk\interface\router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rajat\AppData\Roaming\Python\Python311\site-packages\wandb\sdk\interface\router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-05 09:12:14,322 INFO    MsgRouterThr:25332 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
2025-03-05 09:12:15,831 INFO    Thread-14 (_run_job):25332 [wandb_run.py:_finish():2112] finishing run da24m014-iit-madras/Assignment - 1/1hy195yf
